lib/passport-azure-ad/saml.js

```diff
@@ -284,22 +288,22 @@ SAML.prototype.validateResponse = function (samlResponse, callback) {
         next(null);
       }
     },
-    function(next){
+    function(next) {
       // check to see if we have loaded the x509 certs from the AAD metadata url
-      if(!self.metadata.saml || self.metadata.saml.certs.length === 0) {
+      if (!self.metadata.saml || self.metadata.saml.certs.length === 0) {
```

Comment:
if (!self.metadata.saml || !self.metadata.saml.certs || self.metadata.saml.certs.length === 0)

in case certs is null